Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -119,7 +119,7 @@ catch
     throw
 }
 
-if($isPremiumStorage)
+if($isPremiumStorage -and $additionalArguments -notLike "*/BlobType:page*")
```

Comment:
Increment the patch version for the task in task.json.

Reply:
Done